src/Extensions/AzureBlobPayloads/PayloadStore/IPayloadStore.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.DurableTask;
+
+/// <summary>
+/// Abstraction for storing and retrieving large payloads out-of-band.
+/// </summary>
+public abstract class IPayloadStore
```

Comment:
Since this is not an interface anymore, prefixing it with "I" is probably not a good idea. Consider renaming it to `PayloadStoreBase` or `AbstractPayloadStore` then. (Personally, I wouldn't even mind keeping it an interface to begin with, my only problem is that the current name breaks the convention.)